dll/win32/kernel32/winnls/nls/ptg.nls

```diff
@@ -101,12 +101,12 @@ STRINGTABLE LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
   LOCALE_SDECIMAL ","
   LOCALE_SENGCOUNTRY "Portugal"
   LOCALE_SENGCURRNAME "Euro"
-  LOCALE_SENGLANGUAGE "Portuguese"
+  LOCALE_SENGLANGUAGE "Português"
```

Comment:
These two shouldn't be translated either. Translated ones are `LOCALE_SNATIVELANGNAME` and `LOCALE_SNATIVEDISPLAYNAME `.
```suggestion
  LOCALE_SENGLANGUAGE "Portuguese"
```